src/StackAdmin/Azs.Compute.Admin/Tests/src/Common.ps1

```diff
@@ -1,23 +1,21 @@
 ﻿
+$global:SkippedTests = @(
+    'TestListInvalidLocation',
```

Comment:
@deathly809 why are we skipping these tests?

Reply:
@bganapa Has more information.

Reply:
Not sure why these are skipped , checking

Reply:
The test is not a valid one. If an invalid location is provided , the Compute Admin RP  would default to the existing first one. This is be their design. So the test is being skipped